ibis/backends/flink/__init__.py

```diff
@@ -412,7 +446,19 @@ def create_view(
         Table
             The view that was created.
         """
-        raise NotImplementedError
+        if obj is None:
+            raise exc.IbisError("The obj parameter is required")
+
+        if isinstance(obj, ir.Table):
+            # TODO(chloeh13q): implement CREATE VIEW for expressions
+            raise NotImplementedError
+
+        if overwrite and self.list_views(name):
```

Comment:
```suggestion
        if overwrite and name in self.list_views():
```
I find this a little more readable